spark/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM jupyter/pyspark-notebook:spark-3.1.1
+LABEL maintainer "peloton@lal.in2p3.fr"
+ENV REFRESHED_AT 2021-06-08
+
+# Add repo
+WORKDIR /home/jovyan/work
+ADD . .
+ENV PYTHONPATH=$PYTHONPATH:/home/jovyan/work/python:${SPARK_HOME}/python/test_coverage
+ENV PYTHONPATH=$PYTHONPATH:${SPARK_HOME}/python:${SPARK_HOME}/python/lib/py4j-0.10.9-src.zip
+ENV COVERAGE_PROCESS_START=/home/jovyan/work/.coveragerc
+
+USER root
+RUN apt-get update \
+     && apt-get install -y vim
```

Comment:
Please cleanup the apt caches after installing, see  https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Oops, thanks!

Reply:
https://github.com/escape2020/school2021/pull/55/commits/2578256c55b16bec426146e48f684ffe4f14e8af